### PR TITLE
Adding Selector headers on Prepare and Corruption Risk Menus

### PR DESCRIPTION
**Pull request by stevenndungu**
@africanmathsinitiative/developers  this is ready for review

### CODE REVIEW

**Comment by stevenndungu** [comment]
@dannyparsons  this is ready for you..

File: instat/DlgUseDate.vb [hunk 1]
```diff
@@ -49,6 +49,7 @@ Public Class dlgUseDate
         ucrReceiverUseDate.SetMeAsReceiver()
         ucrReceiverUseDate.bUseFilteredData = False
         ucrReceiverUseDate.SetIncludedDataTypes({"Date"})
+        ucrReceiverUseDate.strSelectorHeading = "Date(s)"
```
Comment (dannyparsons) [inline instat/DlgUseDate.vb]: "Dates" is better. Because we don't use "Variable(s)" or "Factor(s)" on other dialogs

File: instat/dlgSplitText.vb [hunk 2]
```diff
@@ -44,6 +44,7 @@ Public Class dlgSplitText
         ucrReceiverSplitTextColumn.SetMeAsReceiver()
         ucrReceiverSplitTextColumn.bUseFilteredData = False
         ucrReceiverSplitTextColumn.SetIncludedDataTypes({"factor", "character"})
+        ucrReceiverSplitTextColumn.strSelectorHeading = "Factors"
```
Comment (dannyparsons) [inline instat/dlgSplitText.vb]: "Character" is more appropriate for these dialogs

File: instat/dlgMakeDate.vb [hunk 3]
```diff
@@ -281,6 +281,16 @@ Public Class dlgMakeDate
         bUseSelectedColumn = True
     End Sub
 
+    Private Sub SelectorHeader()
+        If rdoDefaultFormat.Checked Then
+            ucrReceiverForDate.strSelectorHeading = "Factors"
+        ElseIf rdoSpecifyOrigin.Checked
+            ucrReceiverForDate.strSelectorHeading = "Numerics"
```
Comment (lilyclements) [inline instat/dlgMakeDate.vb]: Among the software we've been using the term "Variates" not "Numerics". How come you're using "Numerics" here?

File: instat/DlgUseDate.vb (hunk 1 above)
Comment (stevenndungu) [inline instat/DlgUseDate.vb]: yes, i agree with this. changed it.

File: instat/dlgMakeDate.vb (hunk 3 above)
Comment (stevenndungu) [inline instat/dlgMakeDate.vb]: @lilyclements All other dialogues are using "Numerics" and not "Variates". Check #3434

**Comment by stevenndungu** [comment]
this is ready for re-review...changes suggested made.

**Pull request merged**